controllers/discordactions.js

```diff
@@ -0,0 +1,128 @@
+const { INTERNAL_SERVER_ERROR } = require("../constants/errorMessages");
+const admin = require("firebase-admin");
+const config = require("config");
+const jwt = require("jsonwebtoken");
+const discordRolesModel = require("../models/discordactions");
+
+/**
+ * Creates a role
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const DISCORD_BASE_URL = config.get("services.discordBot.baseUrl");
+
+const createGroupRole = async (req, res) => {
+  try {
+    const rolename = `group-${req.body.rolename}`;
+
+    const { wasSuccess } = await discordRolesModel.isGroupRoleExists(rolename);
+
+    if (!wasSuccess) {
+      return res.status(400).json({
+        message: "Role already exists!",
+      });
```

Comment:
what if it fails for some other reason, the message being sent is for HTTP status 409
are we sending 400 irrespective of the reason?